cypress/e2e/comments.spec.ts

```diff
@@ -153,8 +208,6 @@ describe("Comments", () => {
     describe("not a site member", () => {
       before(() => {
         removeOtherCollaborators()
-      })
-      beforeEach(() => {
```

Comment:
why remove this? if we do some permission changes in the test, this change makes it so that the permission changes will propagate to other tests in the same suite

Reply:
The second test for `not a site member` does `setUserAsUnauthorised()` twice. Moving it to test set up instead.